pkgs/development/tools/vala-language-server/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkgconfig
+, scdoc
+, glib
+, libgee
+, json-glib
+, jsonrpc-glib
+, vala
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vala-language-server";
+  version = "0.48.1";
+  repoName = pname;
+
+  src = fetchFromGitHub {
+    owner = "benwaffle";
+    repo = repoName;
+    rev = version;
+    sha256 = "12k095052jkvbiyz8gzkj6w7r7p16d5m18fyikl48yvh5nln8fw0";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    scdoc
+  ];
+
+  buildInputs = [
+    glib
+    libgee
+    json-glib
+    jsonrpc-glib
+    vala
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Code Intelligence for Vala & Genie";
+    homepage = "https://github.com/benwaffle/vala-language-server";
+    license = licenses.lgpl21;
+    maintainers = [ "Felix Andreas <fandreas@physik.hu-berlin.de>" ];
```

```suggestion
    maintainers = with maintainers; [ andreasfelix ];
```

and add yourself to tha maintainers file.

---

thank you! i have implemented you suggestions.